dataland-frontend/tests/e2e/fixtures/common/DataSourceFixtures.ts

```diff
@@ -11,7 +11,7 @@ export function generateDataSource(referencedReports: ReferencedDocuments): Exte
   const chosenReport = pickOneElement(Object.keys(referencedReports));
   const chosenReportReference = referencedReports[chosenReport];
   return {
-    page: faker.number.int({ min: 1, max: 1200 }),
+    page: String(faker.number.int({ min: 1, max: 1200 })),
```

Comment:
It would be great if we also generated random page ranges along with single page references.

Reply:
Working on it

Reply:
Done